Kickstarter-iOS/Views/Controllers/DeprecatedRewardPledgeViewController.swift

```diff
@@ -621,8 +581,8 @@ internal final class DeprecatedRewardPledgeViewController: UIViewController {
 
     for (idx, item) in allItems.enumerated() {
       let label = UILabel()
-        |> UILabel.lens.font .~ (idx == 0 ? .ksr_headline(size: 13) : .ksr_body(size: 14))
-        |> UILabel.lens.textColor .~ (idx == 0 ? .ksr_soft_black : .ksr_text_dark_grey_500)
+        |> UILabel.lens.font .~ (idx == 0 ? UIFont.ksr_caption1(size: 16).bolded : .ksr_body())
```

Comment:
I think for the `Includes` title label the font should be the same as `Description` and `Estimated Delivery` which are both `UIFont.ksr_callout().bolded`.